.github/workflows/cover-ci.yml

```diff
@@ -0,0 +1,41 @@
+name: coverage
+
+on: [push, pull_request]
+
+jobs:
+  run-coverage:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Add conda to system path
+      run: |
+        # $CONDA is an environment variable pointing to the root of the miniconda directory
+        echo $CONDA/bin >> $GITHUB_PATH
+    - name: Install conda env & dependencies
+      run: |
+       conda install pytorch torchvision torchaudio cudatoolkit=11.3 -c pytorch
```

Comment:
I copied it from torch's official website.

Reply:
Maybe remove it in the next commit just to speed up builds a bit. It's not a big deal!

Reply:
Also, since we're running all tests on CPU, I'm pretty sure we don't need cudatoolkit (unless github-actions servers have GPUs?).
I think removing cudatoolkit shoild save is at least a minute.

Reply:
Ok, I will try that.